activesupport/lib/active_support/lazy_load_hooks.rb

```diff
@@ -63,7 +63,7 @@ def with_execution_control(name, block, once)
         end
       end
 
-      def execute_hook(base, options, block)
+      def execute_hook(name, base, options, block)
         with_execution_control(name, block, options[:run_once]) do
```

Comment:
Since we're `extend`ing this module, `name` here likely just referenced the module name — and thus didn't blow up with a `NoMethodError`.